.swiftlint.yml

```diff
@@ -115,38 +50,40 @@ whitelist_rules:
   # Lines should not have trailing whitespace.
   - trailing_whitespace
 
-  # Type bodies should not span too many lines.
-  # - type_body_length
-
-  # Type name should only contain alphanumeric characters, start with an
-  # uppercase character and span between 3 and 40 characters in length.
-  # - type_name
-
-  # Documented declarations should be valid.
-  # - valid_docs
+  - vertical_whitespace
```

Comment:
I guess if we also add `leading_whitespace` we should be able to easily autofix them?